### PR TITLE
Update README so building and deploying test case doesn't fail

### PR DESCRIPTION
**Pull request by davebiagioni**
Instructions for building and deploying a test case -- which say to define the `TESTCASE` env var as a directory rather than the testcase name -- currently fail:

```
$ TESTCASE=./testcases/testcase1/ docker-compose up
[+] Running 1/1
 ⠿ Container project1-boptest-boptest-1  Recreated                                             0.2s
Attaching to project1-boptest-boptest-1
project1-boptest-boptest-1  | Traceback (most recent call last):
project1-boptest-boptest-1  |   File "restapi.py", line 63, in <module>
project1-boptest-boptest-1  |     case = TestCase()
project1-boptest-boptest-1  |   File "/home/developer/testcase.py", line 49, in __init__
project1-boptest-boptest-1  |     self.data_manager.load_data_and_jsons()
project1-boptest-boptest-1  |   File "/home/developer/data/data_manager.py", line 374, in load_data_and_jsons
project1-boptest-boptest-1  |     z_fmu = zipfile.ZipFile(self.case.fmupath, 'r')
project1-boptest-boptest-1  |   File "/usr/lib/python2.7/zipfile.py", line 779, in __init__
project1-boptest-boptest-1  |     self.fp = open(file, modeDict[mode])
project1-boptest-boptest-1  | IOError: [Errno 21] Is a directory: 'models/wrapped.fmu'
project1-boptest-boptest-1 exited with code 1
```

But the [instructions in the user manual](https://ibpsa.github.io/project1-boptest/docs-userguide/getting_started.html#deploy-a-test-case) appear to work.  This PR simply updates the README with these modified steps (if this is indeed an error).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,14 +22,15 @@ Visit the [BOPTEST Home Page](https://ibpsa.github.io/project1-boptest/) for mor
...
-3) Build and deploy a test case using the following commands executed in the root directory of this repository and where <testcase_dir_name> is the name of the test case subdirectory located in [/testcases](https://github.com/ibpsa/project1-boptest/tree/master/testcases):
+3) To build and deploy a test case, use the following commands within the root directory of the extracted software:
 
-  * Linux or macOS: ``$ TESTCASE=<testcase_dir_name> docker-compose up``
-  * Windows PowerShell: ``> ($env:TESTCASE="<testcase_directory>") -and (docker-compose up)``
+  * Linux or macOS: ``$ TESTCASE=<testcase_name> docker-compose up``
+  * Windows PowerShell: ``> ($env:TESTCASE="<testcase_name>") -and (docker-compose up)``
   * A couple notes:
-    * The first time this command is run, the image ``boptest_base`` will be built.  This takes about a minute.  Subsequent usage will use the already-built image and deploy much faster.
+    * Replace ``<testcase_name>`` with the name of the test case you wish to deploy.
```
Comment (dhblum) [inline README.md:30]: Suggest adding where a user can find the test case names.  So maybe something like, "Replace ``<testcase_name>`` with the name of the test case you wish to deploy.  Test case names can be found at https://github.com/ibpsa/project1-boptest/tree/master/testcases or on the web page at https://ibpsa.github.io/project1-boptest/testcases/index.html."

Comment (dhblum) [reply README.md:30]: And maybe replace the raw link text with words like, "...can be found in the [/testcases](link) directory of the repo or in the [Test Cases](link) tab of the web page."

**Review by dhblum — COMMENTED** [review]
Thanks @davebiagioni for the PR!  I agree it'd be good to update this text.  Just take a look at my one inline suggestion.  Also, I'd like to fix some unit test problems in https://github.com/ibpsa/project1-boptest/issues/450 first before merging this.

File: README.md (hunk 1 above)
Comment (davebiagioni) [reply README.md:30]: yep, seems reasonable to me!

Comment (dhblum) [reply README.md:30]: Ok if I write directly to your fork branch to update this PR accordingly?

Comment (davebiagioni) [reply README.md:30]: please do

**Review by dhblum — APPROVED** [review]
This is good to go.  Thanks again @davebiagioni!

**Pull request merged**